docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -76,6 +76,10 @@ Release notes: Open GEE 5.3.3
            - A memory management bug in XML processing was fixed,
              resulting in lower memory utilization and significantly improved
              performance.
+         * - 1577

```

Comment:
1577 is a PR number, not an issue number. Does that matter?

Reply:
It looked to me that some were done by pr already, like 1517 a bit earlier, but it is a good question.  Maybe @tst-rwildes can answer.